pkg/apiserver/rest/dualwriter_mode3.go

```diff
@@ -38,3 +39,24 @@ func (d *DualWriterMode3) Create(ctx context.Context, obj runtime.Object, create
 	}
 	return created, nil
 }
+
+// Get overrides the default behavior of the Storage and retrieves an object from Unified Storage
```

Comment:
This behavior is closer to mode 2 than 3. 

Mode 2 should be that we read from LegacyStorage and enrich the object with fields from Storage. To make it simpler, I have since proposed that we read from Storage where possible and fall back to Legacy otherwise.

For Mode 3 and 4, we read only from Storage.